inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java

```diff
@@ -859,6 +885,22 @@ private boolean hasImplicitNamespace(String s)
         }
         return false;
     }
+    
+    public boolean isIdentiferBaseProperty(KBHandle handle)
```

Comment:
This method doesn't seem to be used anywhere. I also think it doesn't do any good because it doesn't consider the IRIs configured for the KnowledgeBase.

Reply:
Yeah, this method was meant to identify the base IRIs and check if the configured KnowledgeBase IRI matches the base one.